generators/java-v2/sdk/src/SdkGeneratorCli.ts

```diff
@@ -49,15 +52,32 @@ export class SdkGeneratorCLI extends AbstractJavaGeneratorCli<SdkCustomConfigSch
 
     protected async generate(context: SdkGeneratorContext): Promise<void> {
         if (context.config.output.snippetFilepath != null) {
+            const snippetFilepath = context.config.output.snippetFilepath;
+            let endpointSnippets: Endpoint[] = [];
+            try {
+                endpointSnippets = await this.generateSnippets({ context });
+            } catch (e) {
+                context.logger.warn("Failed to generate snippets, this is OK.");
+            }
+
             try {
-                const endpointSnippets = await this.generateSnippets({ context });
                 await this.generateReadme({
                     context,
                     endpointSnippets
                 });
             } catch (e) {
                 context.logger.warn("Failed to generate README.md, this is OK.");
             }
+
+            try {
+                await this.generateSnippetsJson({
+                    context,
+                    endpointSnippets,
+                    snippetFilepath
+                });
+            } catch (e) {
+                context.logger.warn("Failed to generate snippets.json, this is OK");
```

Comment:
`this is OK` is a bit of a weird log (ik our python generator does that). thoughts on excluding it?